Sources/MapboxDirections/Intersection.swift

```diff
@@ -162,11 +165,33 @@ extension Intersection: Codable {
         case tunnelName = "tunnelName"
         case isUrban = "is_urban"
         case restStop = "rest_stop"
-        case administrationRegionIndex = "admin_index"
+        case administrativeRegionIndex = "admin_index"
         case geometryIndex = "geometry_index"
     }
     
+    static func encode(intersections: [Intersection], to parentContainer: inout UnkeyedEncodingContainer, administrativeRegionIndices: [Int?]?) throws {
```

Comment:
I am not particularly happy with this `inout` parameter, but this is how encoding array of objects is done. Passing `Encoder` here is not an option, as we need to call `container.superEncoder()` for each item we encode. Otherwise it just overwrites the JSON node.